### ISSUE TITLE
Models classificació binària

### ISSUE THREAD

**Issue by edurnealvarezz** [edurnealvarezz/TFG#2]
## Context
Models de classificació sobre la variable objectiu

## Objectiu
Entrenar i comparar models

## Checklist
- [x] Logit binari
- [x] Random forest + SHAP
- [x] XGBoost + SHAP
- [ ] Comparació de models
- [ ] Variables importants
- [ ] Millora del millor model

## Output esperat
- Mètriques (accuracy, AUC)
- Interpretació
